### ISSUE TITLE
Qsyncthingtray makes OSX switch to dedicated graphics

### ISSUE THREAD

**Issue by lenucksi** [sieren/QSyncthingTray#210]
Once I launch Qsyncthingtray on my Mac using OS X 10.12 the graphics subsystem is switched to the dedicated graphics card with Qsyncthingtray listed as dependency (reason) for it.
This should not happen.

**Comment by sieren** [comment]
Interesting, are you using the most recent version?
We had fixed this back then as far as I know: https://github.com/sieren/QSyncthingTray/issues/94

**Comment by lenucksi** [comment]
I guess so, it does not find any updates and is the most recent version coming out of Homebrew.

**Comment by sieren** [comment]
Yeah, seems like it is the most recent version. I'll have to take a look when I'm on a different mac with 2 graphics cards.

Does this happen all the time or only when you open the browser window?

**Comment by lenucksi** [comment]
It happens as soon as the program starts / has it's icon appear in the menubar.

**Comment by dodekeract** [comment]
Can't reproduce on a 2017 15" MacBook Pro (max. CPU/GPU/SSD) on `macOS Sierra 10.12.5 (16F2073)`.

**Comment by emarj** [comment]
I'm also experiencing this issue while running macOS High Sierra on MBP Mid-2010.

By the way thanks for this useful software!
